src/coresyf_image_crop/tests/test_image_crop.py

```diff
@@ -18,6 +19,8 @@ def setUp(self):
             "-54172.937265817548905 107225.478697661776096 0," \
             "-61575.57812574980926 107225.478697661776096 0,"\
             "-61575.57812574980926 103754.81329901740537 0))"
+        
```

Comment:
Perhaps remove this blank line here.

Reply:
Due to moving two lines from test_crop_raster, the blank line now makes more sense to be here.